scene/3d/camera_3d.cpp

```diff
@@ -496,27 +496,76 @@ Vector<Vector3> Camera3D::get_near_plane_points() const {
 	return points;
 }
 
-Point2 Camera3D::unproject_position(const Vector3 &p_pos) const {
-	ERR_FAIL_COND_V_MSG(!is_inside_tree(), Vector2(), "Camera is not inside scene.");
+bool Camera3D::safe_unproject_position(const Vector3 &p_pos, Point2 &r_result) const {
+	ERR_FAIL_COND_V_MSG(!is_inside_tree(), false, "Camera is not inside scene.");
```

Comment:
Returning `false` in error case will also cause the `WARN_PRINT_ONCE` to print, which results in a confusing message.

Reply:
Ah good point! I'll fix this. :+1: